src/routes/templates/templates.json

```
@@ -836,5 +836,19 @@
 		"category": "Svelte",
 		"tags": ["templates"],
 		"stars": 0
+	},
+	{
+   		 "title": "Svelte-Typescript-Routify-Tailwindcss-template",
+  		 "url": "https://github.com/skshahriarahmedraka/Svelte-Routify-ts-tailwindcss-template.git",
+		 "description": "Svelte & Typescript & Routify & Tailwindcss template",
+ 		 "npm": "",
+ 		 "addedOn": "2022-05-07",
+  		 "category": "Svelte",
+  		 "tags": [
+  		     "templates",
+   		     "typescript",
+  		     "integrations"
+   		 ],
+  		 "stars": 0
```

This file use tabulation only.
You can run `npm run format` to fix all the formatting errors

```suggestion
   		"title": "Svelte-Typescript-Routify-Tailwindcss-template",
  		"url": "https://github.com/skshahriarahmedraka/Svelte-Routify-ts-tailwindcss-template.git",
		"description": "Svelte & Typescript & Routify & Tailwindcss template",
 		"npm": "",
 		"addedOn": "2022-05-07",
  		"category": "Svelte",
  		"tags": [
  			"templates",
   			"typescript",
  			"integrations"
   		],
  		"stars": 0
```